.gitmodules

```diff
@@ -1,6 +1,6 @@
-[submodule "osu-framework"]
-	path = osu-framework
-	url = https://github.com/ppy/osu-framework
-[submodule "osu-resources"]
-	path = osu-resources
-	url = https://github.com/ppy/osu-resources
+[submodule "osu-framework"]
+	path = osu-framework
+url=https://github.com/Memorix101/osu-framework
+[submodule "osu-resources"]
+	path = osu-resources
+	url = https://github.com/ppy/osu-resources
```

Comment:
pretty sure it should be in the repo? it's required to resolve the submodules to a repository.

just don't commit it.